### DISCUSSION TITLE
Breaking change in gptel: gptel-models as symbols

### DISCUSSION THREAD

**Discussion by karthink** [daedsidog/evedel#6] [General]
Just a heads-up that in preparation for media/images, function calling and json support in gptel, I now require `gptel-model` to be a symbol, i.e. `(setq gptel-model 'gpt-4o-mini)` instead of `(setq gptel-model "gpt-4o-mini")`.

This is a breaking change but I'm trying to paper over this difference: specifying `gptel-model` as a string should still work fine, but with a warning.  Sorry for the trouble!

This will be effective in gptel version 0.9.5, which I'll be tagging today.  In a future version the warning will be upgraded to an error, hopefully giving everyone enough time to make the change.

**Comment by daedsidog** [comment]
Thanks for the heads up. Fortunately I only use gptel's `gptel-request` to make calls for model queries right now, so I don't think I need to do anything on my end.
